doc/api/net.md

```diff
@@ -941,7 +941,7 @@ For TCP connections, available `options` are:
 * `autoSelectFamilyAttemptTimeout` {number}: The amount of time in milliseconds to wait
   for a connection attempt to finish before trying the next address when using the `autoSelectFamily` option.
   If set to a positive integer less than `10`, then the value `10` will be used instead.
-  **Default:** `250`.
+  **Default:** initially `250`, but it can be changed at runtime using [`net.setDefaultAutoSelectFamilyAttemptTimeout(value)`][]
```

Comment:
```suggestion
  **Default:** [`net.getDefaultAutoSelectFamilyAttemptTimeout()`][]
```

Reply:
I'm a bit conflicted on this. While it might make sense to reference the getter, I phrased like this (and similarly for the `autoSelectFamily` option) to immediately give user a sense of the default, rather than having them jump back and forth in the various sections.

Reply:
It's an optional suggestion, I don't have a strong opinion 🙂 I would argue that the default _should not matter_ to most people coming across this section because enabling `autoSelectFamily` by default is only a good idea if it _just works_. If developers frequently have to change parameters, that's not a good sign. Thus, I would prefer the shorter variant.

(If this setting was a property, I would suggest to just put the default value there. Given that it's an explicit setter and getter, I don't know where to put the default. Maybe in the getter description?)

Reply:
Well, after this PR I'll send another PR which enables `autoSelectFamily` by default and parameters will stay most of the time the default.

You know what? I'll go for the shorter sentences for both options, they are easier to read.
I'll leave as this in this PR and change in the other PR so I don't forget. Is this fine?

Reply:
Of course, as I said, it's an optional suggestion. I'm happy with either :)

Reply:
Done in #46790.